packages/uicore/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@harness/uicore",
-  "version": "3.80.2",
+  "version": "3.81.2",
```

You'll need to reset the patch version to 0
```suggestion
  "version": "3.81.0",
```